### DISCUSSION TITLE
Does TOP LANGUAGES card include private repositories

### DISCUSSION THREAD

**Discussion by CPang23** [anuraghazra/github-readme-stats#1822] [Q&A] [answered]
couldn't find anything on it from the internet so decided to ask here

**Comment by justin-2028** [comment, marked as answer]
In short, no, but you can deploy your own instance to do so.

Refer to this: https://github.com/anuraghazra/github-readme-stats/issues/653

Hope that helps

**Reply by CPang23** [reply]
ill check it out
